terms/crossvalidation_lmd.tex

```diff
@@ -75,7 +75,7 @@ \chapter{\textit{}Cross-Validation}
 \label{fig:models}
 \end{figure*}
 
-As stated in \cite{Kohavi1995study}, the LOO method gives a 0\% accuracy on the test set when the number of target labels are equal to the number of instances in the dataset. It is shown that the $k$-fold CV method gives much better results, due to its lower variance, especially when $k = {10, 20}$. Furthermore, R. Kohavi et al. state that the best accuracy is achieved when using the stratified cross-validation method, since this has the least bias.
+As stated in \cite{kohavi1995study}, the LOO method gives a 0\% accuracy on the test set when the number of target labels are equal to the number of instances in the dataset. It is shown that the $k$-fold CV method gives much better results, due to its lower variance, especially when $k = {10, 20}$. Furthermore, R. Kohavi et al. state that the best accuracy is achieved when using the stratified cross-validation method, since this has the least bias.
```

Comment:
I updated it to fix a wrong case warning.

Reply:
thanks